distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Time_Period.enso

```diff
@@ -33,3 +34,10 @@ type Time_Period
     adjust_end : (Time_Of_Day | Date_Time) -> (Time_Of_Day | Date_Time)
     adjust_end self date =
         (Time_Utils.utils_for date).end_of_time_period date self.to_java_unit
+
+    to_duration : Duration
```

Comment:
This method needs _some_ comment.

Either a docstring if it's meant for public use or `## PRIVATE` to hide it.

Reply:
I made it private; it seems like this is meant mainly for specifying units and not as a kind of duration, mostly.